jetty-client/src/test/java/org/eclipse/jetty/client/HttpClientProxyProtocolTest.java

```diff
@@ -173,8 +174,8 @@ public void testClientProxyProtocolV2WithVectors() throws Exception
             protected void service(String target, Request jettyRequest, HttpServletRequest request, HttpServletResponse response) throws IOException
             {
                 EndPoint endPoint = jettyRequest.getHttpChannel().getEndPoint();
-                assertTrue(endPoint instanceof ProxyConnectionFactory.ProxyEndPoint);
-                ProxyConnectionFactory.ProxyEndPoint proxyEndPoint = (ProxyConnectionFactory.ProxyEndPoint)endPoint;
+                assertTrue(endPoint instanceof ProxyEndPoint);
+                ProxyEndPoint proxyEndPoint = (ProxyEndPoint)endPoint;
```

Comment:
Breaking change, can you revert?

Reply:
Yes. Next push will have this class back where it originally was.